src/EnergyPlus/CondenserLoopTowers.hh

```diff
@@ -92,21 +92,21 @@ namespace CondenserLoopTowers {
 
     enum class PIM
     {
-        Unassigned,
+        Unassigned = -1,
         NominalCapacity,
         UFactor
     };
 
     enum class CapacityCtrlEnum
     {
-        Unassigned,
+        Unassigned = -1,
```

Comment:
I don't really care if we call `enum class` types `X` or `XType` or `XEnum` or `X_T` or `X_E` (`XTypeEnum` is bad because an `enum class` *is* a type), but we should do it consistently.